packages/qdrant-loader/src/qdrant_loader/connectors/git/connector.py

```
@@ -405,25 +414,20 @@ async def get_documents(self) -> list[Document]:
-            documents = []
-
             for file_path in files:
                 if not self.file_processor.should_process_file(file_path):  # type: ignore
                     continue
 
                 try:
                     document = self._process_file(file_path)
-                    documents.append(document)
+                    yield document
 
                 except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)